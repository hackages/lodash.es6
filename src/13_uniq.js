/**
 * Creates a duplicate-free version of an array
 */
export function uniq(collection) {}
