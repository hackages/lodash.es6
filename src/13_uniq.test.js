import {uniq} from './13_uniq'

describe('uniq', () => {
  it('should remove duplicate', () => {
    expect(uniq([1, 2, 3, 4])).toMatchSnapshot()
    expect(uniq([1, 1])).toMatchSnapshot()
    expect(uniq([null, null, true, true])).toMatchSnapshot()
    expect(uniq([{a: 32}, {a: 32}, {a: 32, b: 21}])).toMatchSnapshot()
  })
})
