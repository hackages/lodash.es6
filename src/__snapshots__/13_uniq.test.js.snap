// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uniq should remove duplicate 1`] = `
Array [
  1,
  2,
  3,
  4,
]
`;

exports[`uniq should remove duplicate 2`] = `
Array [
  1,
]
`;

exports[`uniq should remove duplicate 3`] = `
Array [
  null,
  true,
]
`;

exports[`uniq should remove duplicate 4`] = `
Array [
  Object {
    "a": 32,
  },
  Object {
    "a": 32,
  },
  Object {
    "a": 32,
    "b": 21,
  },
]
`;
